vault-change-publisher/scripts/commit-reviewed-publication.py

```
@@ -416,14 +488,208 @@ def stable_regular_bytes(path: Path) -> bytes:
+                    parent_descriptor,
+                    path.name,
+                )
+                quarantined = False
+                raise CommitError(
+                    "newly installed artifact changed during rollback; rollback refused"
+                )
+            except FileExistsError:
+                raise CommitError(
+                    "newly installed artifact changed during rollback; "
+                    "rollback quarantined it"
+                )
```

Ruff (0.16.1)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)